Sources/SwiftBluetooth/Peripheral/Peripheral+DiscoveryInfo.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+import CoreBluetooth
+
+extension Peripheral {
+    public struct DiscoveryInfo {
+        public var RSSI: Int
```

Comment:
The convention is to use lowercase, even for acronyms: https://developer.apple.com/documentation/corelocation/clbeacon/1621557-rssi#

Reply:
I was following the arguments used in [central manager delegate](https://developer.apple.com/documentation/corebluetooth/cbcentralmanagerdelegate/1518937-centralmanager) and [peripheral delegate](https://developer.apple.com/documentation/corebluetooth/cbperipheraldelegate/1620304-peripheral).

Glad they've set the precedent of using lowercase somewhere else, I honestly assumed it was some convention with RSSI specifically.